src/System.ServiceModel.Http/src/System.ServiceModel.Http.csproj

```diff
@@ -5,7 +5,7 @@
     <AssemblyName>System.ServiceModel.Http</AssemblyName>
     <AssemblyVersion>4.1.0.0</AssemblyVersion>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
```

Comment:
S.SM.Http is not a partial façade either. Actually none of our 5 contracts is partial façade. Not sure what impact this property has to the build, so just to point out.